### DISCUSSION TITLE
Loading workflows at startup

### DISCUSSION THREAD

**Discussion by RussellTaylor83** [orkes-io/orkes-conductor-community#15] [Q&A]
Hello,

We've been running conductor for a few years, but Orkes community is looking like a much better option (primarily due to the documentation!).

What are people doing to load workflows at orkes pod startup?

With conductor we poll the api until it's available, then a shell script posts all our workflows from within the container. This is somewhat janky, is there a correct way to do it please?

Thanks!

Russ

**Comment by manan164** [comment]
Hi @RussellTaylor83 , Thanks for bringing this up. There are several ways using which we can load workflow definitions.
The one which you suggested also works fine but the issue is, shell scripting is error prone mechanism. If you can via either python or use GitHub pipeline load workflow definitions then that will provide a better audit mechanism. 

Happy to jump on a call for this or we can chat [here](https://join.slack.com/t/orkes-conductor/shared_invite/zt-xyxqyseb-YZ3hwwAgHJH97bsrYRnSZg) for more realtime collaboration cc @v1r3n

**Reply by RussellTaylor83** [reply]
Thank you, I've jumped onto your slack to discuss. I guess the problem is persistence when the pod restarts, and unfortunately we have no access to Github where this is deployed.
